js/logo.js

```diff
@@ -257,6 +258,7 @@ function Logo () {
     this.neighborArgNote2 = {};
     this.neighborArgBeat = {};
     this.neighborArgCurrentBeat = {};
+    this.frequency;
```

Comment:
does this need to be an array so we can have chords? And does it need to be one per turtle for multiple voices?